Common/Transforms/itkStackTransform.hxx

```diff
@@ -56,13 +56,8 @@ StackTransform<TScalarType, NInputDimensions, NOutputDimensions>::SetParameters(
   const NumberOfParametersType numSubTransformParameters = this->m_SubTransformContainer[0]->GetNumberOfParameters();
   for (unsigned int t = 0; t < this->m_NumberOfSubTransforms; ++t)
   {
-    // MS, \todo: the new itk::TransformParameters only have constructors taking 1 argument
```

Comment:
@mstaring This pull request finally fixes the `\todo` that you wrote many years ago 😃  The new (ITK 5.2) `itk::OptmizerParameters` does have a few more convenient constructors than before, including one that accepts a data pointer and a size as arguments! No need to use a temporary `subarray` anymore!

Reply:
nice work and lgtm!